### DISCUSSION TITLE
Why it's own UWSGI?

### DISCUSSION THREAD

**Discussion by jfmatth** [piku/piku#316] [Ideas]
Any reason we don't use the global UWSGI service for piku?

you could add the piku uwsgi directory in the global config.

Seems like it would be a simpler / more standard config then?

**Comment by mariusa** [comment]
There are also these issues, for the same topic:
https://github.com/piku/piku/issues/265
https://github.com/piku/piku/issues/279

**Comment by rcarmo** [comment]
Yeah, I think we can drop that. I’ve had zero issues with my deployment strategy. @chr15m, can you take a look at the workbooks, etc? I only deploy via cloud-init these days.

**Reply by chr15m** [reply]
It may take me a bit of time to get to this but I'll add an issue so it doesn't get forgotten.
